packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -457,9 +457,13 @@ export enum RuntimeHeaders {
 
 /** True if a tombstoned object should be returned without erroring */
 export const AllowTombstoneRequestHeaderKey = "allowTombstone"; // Belongs in the enum above, but avoiding the breaking change
+/** [IGNORED IF throwOnInactiveLoad OPTION NOT SET] True if an inactive object should be returned without erroring */
```

Comment:
This should be moved to `gcDefinitions`. Eventually the tombstone ones should be too.

Reply:
IMO we should export them both. When we think we don't want to support them anymore we can deprecate/remove them just like anything else.